### ISSUE TITLE
No colors in the default styles

### ISSUE THREAD

**Issue by consal** [gregorias/anki-code-highlighter#71]
**Bug description**
Using the plugin with default styles or any other styles results in no style being applied.

**Reproduction steps**
1. Install the plugin
2. Format a code snippet
3. Only the font changes, no colors are applied.

The HTML of the test card:

```html
<div class="pygments" style="display:flex; justify-content:center;">
<pre><code class="nohighlight"><span class="k">def</span> <span class="nf">main</span><span class="p">():</span>
    <span class="k">pass</span>
</code></pre>
</div>
```

**Expected behavior**
Styling should be applied.

**Screenshots**
![obraz](https://github.com/gregorias/anki-code-highlighter/assets/51994060/1be68705-e559-4eb3-9cbc-cbd335b00e16)

**Card template**
Back template
```html
{{FrontSide}}

<hr id=answer>

{{Tył}}

<!-- Anki Code Highlighter (Addon 112228974) BEGIN -->
<link rel="stylesheet" href="_ch-pygments-solarized.css" class="anki-code-highlighter">
<link rel="stylesheet" href="_ch-hljs-solarized.css" class="anki-code-highlighter">
<script src="_ch-highlight.js" class="anki-code-highlighter"></script>
<script src="_ch-my-highlight.js" class="anki-code-highlighter"></script>
<!-- Anki Code Highlighter (Addon 112228974) END -->

```

**Additional information**
 - OS version: Windows 11
 - Anki version:
  Version 23.10.1 (fac9e0ee)⁩
  Python 3.9.15 Qt 6.6.0 PyQt 6.6.0

**Additional context**
 I tried

- using both renderers
- different programming languages
- refreshing code highlighter assets from the menu
- even copying the css around inside the `%APPDATA%\Anki` directory

**Comment by gregorias** [comment]
Could you fill out [the bug template](https://github.com/gregorias/anki-code-highlighter/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=)? Could you [refresh the plugin](https://github.com/gregorias/anki-code-highlighter#refresh--removal)?

**Comment by consal** [comment]
Hi, sure I'll fill it tomorrow.

I tried now: `Delete Code Highlighter Assets` -> Restart -> `Delete the plugin` -> Restart -> `Install the plugin` -> Restart. 

It's still the same, no styling.

**Comment by consal** [comment]
I updated the issue with the bug template

**Comment by gregorias** [comment]
Thank you for the details.

Could you provide a screenshot of the card's preview? Styles through CSS classes are not applied in the edit mode, so what you've presented so far is expected.

![image](https://github.com/gregorias/anki-code-highlighter/assets/722385/f3b71dca-d24a-426d-9698-a214a93407bf)

**Comment by consal** [comment]
Lol, it works now! I was learning before filling the bug request, and there was no formatting. After filling the bug request, suddenly formatting works in all my cards. I have no idea what fixed it. Must've been one action I did when filling the report. Opening HTML view? Opening card template and saving it again? No idea. I also tested now with different card templates, it worked in all of them.

(By the way, the screenshot I took yesterday and had in the originally opened issue with no formatting was from learning card, not from editing view.)

I couldn't solve it for months, I have hundreds of cards using this extension, and finally I can see some colors! <3
Thanks for your help!

![obraz](https://github.com/gregorias/anki-code-highlighter/assets/51994060/4f1167d8-6b0d-43e4-9cc4-b4b6fb93007a)
